<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BanDialog</class>
 <widget class="QDialog" name="BanDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>419</width>
    <height>142</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Banning user</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Enter reason</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="banReason"/>
   </item>
   <item>
    <widget class="QCheckBox" name="banCertificate">
     <property name="text">
      <string>Ban User Certificate</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="banIP">
     <property name="text">
      <string>Ban IP</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>Ban</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
